### PR TITLE
Add scripts to auto update en.json, strings.json and services.yaml

### PR DESCRIPTION
**Pull request by basnijholt**

### CODE REVIEW

**Review by th3w1zard1 — APPROVED** [review]

File: custom_components/adaptive_lighting/services.yaml [hunk 1]
```diff
@@ -2,251 +2,249 @@ apply:
...
     separate_turn_on_commands:
-      description: "separate_turn_on_commands: Separate the commands for each attribute (color, brightness, etc.) in 'light.turn_on' (required for some lights)."
+      description: Use separate `light.turn_on` calls for color and brightness, needed for some light types. 🔀
       required: false
       example: false
       selector:
-        boolean:
+        boolean: null
     send_split_delay:
-      description: "send_split_delay: wait between commands (milliseconds), when separate_turn_on_commands is used. May ensure that both commands are handled by the bulb correctly."
```
Comment (basnijholt) [inline custom_components/adaptive_lighting/services.yaml:152]: Old description better?

File: custom_components/adaptive_lighting/services.yaml [hunk 2]
```diff
@@ -2,251 +2,249 @@ apply:
...
   fields:
     entity_id:
-      description: "entity_id of the Adaptive Lighting switch."
+      description: Entity ID of the switch. 📝
       required: true
       selector:
         entity:
           domain: switch
     use_defaults:
-      description: "(default: 'current' for current settings) You can set this to 'factory', 'configuration', or 'current' to reset the variables not being set with this service call. 'current' leaves them as is, 'configuration' resets to whatever already initializes at startup, 'factory' resets to the default values listed in the documentation."
```
[the remark this answers is not in the document]
Comment (th3w1zard1) [reply custom_components/adaptive_lighting/services.yaml:76]: So where's the origin for all of your autofill documentation code? Is it expecting an entry in `const.py` or `strings.json`?

File: .github/update-strings.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import json
```
Comment (basnijholt) [inline .github/update-strings.py:1]: Still need to make sure these scripts are called in CI

File: custom_components/adaptive_lighting/services.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (basnijholt) [reply custom_components/adaptive_lighting/services.yaml:76]: Should be updated in `const.py`. Then the other things will auto populate.

Just want to flag all the places where the description might be worse now.

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -0,0 +1,2 @@
+@basnijholt
+@th3w1zard1
```
Comment (basnijholt) [inline .github/CODEOWNERS:2]: I added this file here to make sure that PRs do not accidentally get automerged before clicking on the "Approve" button in the review.

File: .github/update-strings.py (hunk 3 above)
Comment (basnijholt) [reply .github/update-strings.py:1]: done

**Pull request merged**